timestamp.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+
+const app = express();
+
+app.set("view engine", "ejs");
+app.use(express.static("./static"));
+
+app.get("/timestamp/:timeStamp", (req, res) => {
+    const TimeStamp = req.params.timeStamp;
+
+    console.log("timeStamp : "+TimeStamp);
+    let timeStamp = "-";
+
+    if(TimeStamp === "") {
+        timeStamp = (new Date()).getTime();
```

Comment:
also, wouldn't dates differ by few nano-seconds if you initialize new Date on line 15 and 16?